browserstack.browsers.js

```diff
@@ -1,4 +1,15 @@
 const browserstackBrowsers = {
+  // iOS
+  iphone: {
```

Comment:
@joscha yeah i tried that in the iOS simulator and it didn't trigger a new window on Mobile Safari (but worked on desktop browsers)